app/models/katello/concerns/smart_proxy_extensions.rb

```diff
@@ -140,6 +140,10 @@ def pulp3_enabled?
 
       def pulp3_support?(repository)
         type = Katello::RepositoryTypeManager.repository_types[repository.content_type]
+        if type.nil?
```

Comment:
Sorry, I have nothing else to offer but to ask: isn't `return unless type` more idiomatic?

Reply:
An easy change. I'll make it so.